src/bloqade/task/braket.py

```diff
@@ -30,30 +40,78 @@ class BraketTask(RemoteTask):
     )
 
     def submit(self, force: bool = False) -> "BraketTask":
+        """
+        Submits the task to the backend.
+
+        Args:
+            force (bool): Whether to force submission even if the task is already submitted.
+
+        Returns:
+            BraketTask: The current task instance.
+
+        Raises:
+            ValueError: If the task is already submitted and force is False.
+
+        Example:
+            >>> backend = BraketBackend(...)  # Create a backend instance
+            >>> task_ir = QuEraTaskSpecification(...)  # Create a task specification
```

Comment:
like my comment from the other PR, this is too vague on how you construct a specification, the task will be constructed from a submission, so you should first create a submission (a batch object) then return the task from the batch.

Reply:
ref: https://github.com/QuEraComputing/bloqade-python/blob/main/src/bloqade/task/batch.py

and you can also check the corresponding tests to understand how this works